docs/_layouts/default.html

```diff
@@ -56,9 +56,10 @@
             </ul>
           </li>
           <li>
-            <span class="nav-header">Bolt Developer Updates</span>
+            <span class="nav-header">
+              <a href="{{ '/blog' | relative_url }}" class="center-vertical">Bolt Developer Updates</a></span>
             <ul>
-              {% for post in site.posts limit: 3 %}
+              {% for post in site.posts reversed limit: 3 %}
```

Comment:
I don't think we want `reversed`. It seems like `posts` already sort from newest to oldest.